### ISSUE TITLE
Add Grunt as alternative Task-Runner

### ISSUE THREAD

**Issue by danielkoch** [namics/generator-nitro#7]
It would be great if the generator support Grunt as a alternative to Gulp

**Comment by brunschgi** [comment]
Hi Daniel
Due to our strife for exceptional performance we haven't planned Grunt support. What is the intention behind your feature request?

**Comment by danielkoch** [comment]
Hi Remo
Similar to LESS/SCSS it could be possible to choose your prefered task runner in the project setup dialog.
I think with future versions of Grunt the performance issue maybe will be fixed.
With the option to choose you enable the whole grunt ecosystem for your project.

**Comment by deniaz** [comment]
Hey Daniel (and Remo),

I've talked to @Regaddi about a similar issue (handlebars as the built-in template engine for server-side code). IMHO Splendid should be extendable and components should be swappable. On the other hand Splendid should be as easy as Terrific Micro and just *work* out of the box.

Providing too many options has a few cons. The setup process for a new project takes (even) longer and requires decisions to be made which may be irrelevant at this point. Leaving these kind of decisions to the developer will lead to a diverse ecosystem (Project A uses `less` with `Grunt`, Project B `sass` with `Gulp`,  Project C `Stylus` with `Broccoli`). It also leads to a lot more code which has to be maintained in this repository.

In this situation we'd lose the ability for devs to switch projects with as little hassle as possible. I firmly believe one needs to be able to swap components on a per-project basis and that's possible with the way it's implemented now. I've described [here](https://github.com/namics/generator-splendid#using-another-template-engine) how to swap the template engine. Replacing gulp with grunt is not a trivial task. If it's really needed I think it should be in the actual project's scope.

AFAIK introducing Terrific as a concept was and still is an attempt to standardise our frontend development process. I fear bloating Splendid with too many technologies and frameworks might provoke the exact opposite.
